var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -49,10 +49,14 @@ class Mfem(Package, CudaPackage, ROCmPackage):
     # other version.
     version('develop', branch='master')
```

Comment:
Can we rename this to `master`? If you want to be really careful, you can add a new version `master` and deprecate `develop` in case anyone is relying on this.

Reply:
In MFEM, `master` is the development branch, so using "develop" for the version name makes sense -- I think this is the convention used by most spack packages.

Reply:
Most Spack packages with branches use the same name as the branch. Spack supports the following versions, all of which sort as newer than the latest numeric release:

* develop
* main
* master
* head
* trunk

Reply:
I don't see a point in changing this, dependent packages already use `mfem@develop` when they need it.

Reply:
I'm going to merge this PR. This change can be discussed further with maintainers and in case submitted as a separate PR.